backend/hitas/calculations/max_prices/max_price.py

```diff
@@ -157,6 +159,7 @@ def calculate_max_price(
         "valid_until": valid_until,
         "maximum_price": max_price,
         "index": max_index,
+        "new_hitas": new_hitas_rules,
```

Comment:
I prefer using more explicit variable names, e.g.: `use_new_hitas_rules` :)

Reply:
I opted to use `new_hitas` as it's consistent with our filter parameter with a same name :)